server/apps/common/model_utils.py

```diff
@@ -10,7 +10,7 @@ class Meta:
 
 
 class AuditModel(models.Model):
-    created_by = models.CharField(max_length=100)
+    created_by = models.CharField(max_length=100, blank=True, null=True)
     updated_by = models.CharField(max_length=100, blank=True, null=True)
```

Comment:
Can never be blank or null

Reply:
I have removed the null constrains and the default value for `created_by` field is set to 'anonymous'. This is because when we pass the request data to serializer then if the `created_by` field is null then the  serializer validation fails. To make the serializer validation pass, the default value of `created_by` is set the 'anonymous' and in the create method of the TodoSerializer I'm setting the actual value of the created_by field that is the current user.